egs2/chime7_task1/asr1/run.sh

```diff
@@ -80,9 +80,15 @@ asr_batch_size=$(calc_int 128*$ngpu) # reduce 128 bsz if you get OOMs errors
 asr_max_lr=$(calc_float $ngpu/10000.0)
 asr_warmup=$(calc_int 40000.0/$ngpu)
 
-if [ $decode_only -eq 1 ]; then
+if [ $decode_only == "dev" ]; then
```

Comment:
Will you add a check if `decode_only` is neither dev nor eval?
I feel `decode_only` is a bit confusing. How about using `skip_train` (default: false) to  control whether to process the training set? And using `test_sets` to let users define the gss and asr_tt sets? For example
```
test_sets="chime6_dev dipco_dev mixer6_dev"
gss_dsets+="${test_sets}"

test_sets_list=(${test_sets// / })
asr_tt_sets=
for dset in ${test_sets_list[@]}; do
    asr_tt_sets+=$(echo "kaldi/${dset}/gss " | tr "_" "/") 
```

Reply:
This is nice but will require too much change in the codebase at this stage.

Reply:
yeah, indeed. We may not need to update this at the moment.